lib/sequences/06_token_introspection_sequence.rb

```diff
@@ -102,4 +102,46 @@ class TokenIntrospectionSequence < SequenceBase
 
   end
 
+  test 'Refresh Token - Token introspection endpoint responds properly to introspection request',
```

Comment:
We are trying to keep the test names as consistent as possible going forward.  Could you change this (and the other) to 

```Token introspection endpoint responds properly to introspection request for refresh token```

And then add `for access token` to the previous ones that are access-token specific?